### ISSUE TITLE
Unify code style between girder_worker and girder

### ISSUE THREAD

**Issue by cjh1** [girder/girder_worker#105]
This probably means adopting girders style :smile_cat:

**Comment by jeffbaumes** [comment]
Just wanted to bubble this one back up, since this is the time where the new girder_worker code is starting to solidify.

**Comment by cjh1** [comment]
As I raised this, I am happy to take on the conversion if we can decide on a style :smile:

**Comment by kotfic** [comment]
now would also be a good time as there are no major PR's open,  or immediately planned

**Comment by cjh1** [comment]
This there consensus that we should use Girders style?

**Comment by cjh1** [comment]
Is "Girder style" documented any where? All I found was [this](http://girder.readthedocs.io/en/latest/development.html?highlight=style)

**Comment by zachmullen** [comment]
We have a flake8 config file, the only other major difference is that girder uses camel case for historical reasons.

**Comment by cjh1** [comment]
I will start with flake8, do we want CamelCase as well?

**Comment by jeffbaumes** [comment]
I thought it felt somewhat disjointed to have camel and underscore together, e.g. when writing a girder endpoint that uses girder_worker, so I'd vote to unify to camel. I'd want some actual dev input though before deciding. Is camel disliked enough that devs would rather have mixed underscore/camel code (and know what to use in what codebase) instead of going all camel?

**Comment by cjh1** [comment]
The flake8 config currently used by girder_worker is pretty much the same as Girder's ( unsurprisingly )

**Comment by cjh1** [comment]
Personally I prefer snake_case ( its also the convention else where ... ) but I also like consistency.

**Comment by zachmullen** [comment]
snake_case is now the officially sanctioned convention in python, but it would be much more disruptive to change girder to snake_case than to change the worker to camel case, so I'd also vote to do the latter.

**Comment by cjh1** [comment]
May be this could help: https://github.com/abarker/camel-snake-pep8

**Comment by danlamanna** [comment]
> I thought it felt somewhat disjointed to have camel and underscore together.

Agreed it does, and this is how most of Girder interacts with the rest of the Python ecosystem now. For instance just look at when Girder code meets other libraries (cherrypy, requests, pymongo, dogpile, boto, click, etc).

I think leaving GW as is might actually be better here, especially given that most task code is wrangling together other libraries which are almost all snake_case.

**Comment by kotfic** [comment]
this morning i agreed with @cjh1 about consistency over my preference (snake_case) but i think dan brings up an important point.   EIther way we should be ensuring we're consistent throughout girder worker (which i'm not sure we are)  are there flake8 rules we can add that enforce snake_case/camel_case?
